polls/models.py

```diff
@@ -1,3 +1,54 @@
+from tabnanny import verbose
 from django.db import models
+from django.contrib.auth.models import AbstractUser
+from django.contrib.auth import get_user_model
 
-# Create your models here.
+class User(AbstractUser):
```

Comment:
How does this work with the oauth stuff?

Reply:
This basically extends the builtin User table in Django. It has the same role as base User model.